recipe/meta.yaml

```
@@ -10,21 +10,23 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
   disable_pip: true
   entry_points:
     - pip = pip._internal.cli.main:main
     - pip3 = pip._internal.cli.main:main
 
 requirements:
   host:
-    - python >=3.8
+    - python >=3.13.a0           # [not with_setuptools_wheel]
```

Not 100% sure about the dot before a0 and missing patch version; in any case, we only have rc1 builds, so I think this could be
```suggestion
    - python >=3.13.0rc1         # [not with_setuptools_wheel]
```